Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -82,28 +107,24 @@ class ITKIONIFTI_EXPORT NiftiImageIO : public ImageIOBase
 
   //-------- This part of the interfaces deals with reading data. -----
 
-  /** Possible file formats readable by this ImageIO implementation.
-   * Used by DetermineFileType(). */
-  enum FileType
-  {
-    /** 2-file Nifti (consisting of .hdr and .img file). */
-    TwoFileNifti = 2,
-    /** 1-file Nifti (consisting of .nii file). */
-    OneFileNifti = 1,
-    /** Legacy Analyze 7.5 format (consisting of .hdr and .img file). */
-    Analyze75 = 0,
-    /** Some other file format, or file system error. */
-    OtherOrError = -1,
-  };
-
+#if !defined(ITK_LEGACY_REMOVE)
```

Comment:
A similar backwards compatibility block could be added to the namespace scope for `Analyze75Flavor`, see other comment.

Reply:
Looks reasonable. Thanks for the suggestion @dzenanz ! If I am not mistaken, @seanm @vfonov @hjmjohnson might use this class on a regular basis, so commenting on this aspect would be appreciated as well.

Reply:
Done in 807dcef.